src/Vrx_main.cpp

```diff
@@ -168,11 +200,50 @@ void ProcessMSPPacket(mspPacket_t *packet)
     DBGLN("Processing MSP_ELRS_SET_VRX_BACKPACK_WIFI_MODE...");
     RebootIntoWifi();
     break;
+  case MSP_ELRS_BACKPACK_SET_RECORDING_STATE:
```

Comment:
Is there a way just to forward the msp packet without rebuilding?

Reply:
There definitely is.
However I thought it might be a good idea to decode it here for other non-MSP based modules that could potentially use the data in the MSP packets (in a general sense, not just for DVR start/stop). This is the function that farms that data out to the respective "driver" classes, so it seemed like a good place to pull it apart.

Happy to make a plan to only support MSP-serial based interfaces from now on though? In which case directly forwarding it would be fine?